### PR TITLE
added next.js font module, updated fonts, updated login page

### PR DESCRIPTION
**Pull request by rsandb**

### CODE REVIEW

**Comment by yk** [comment]
looks cool so far!
for the logos, there is a new one in the repo top-level `assets` folder. could you use that one? it's about 5 times smaller and doesn't include an unnecessary embedded PNG

**Comment by rsandb** [comment]
Yeah, I can change that! My fork wasn't synced when that was added so I'll do that

**Review by yk — APPROVED** [review]
lgtm, thank you!

File: website/src/pages/auth/signup.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                leftIcon={<FaEnvelope />}
+                colorScheme="gray"
+                onClick={signinWithEmail}
+                // isDisabled="false"
+              >
+                Continue with Email
+              </Button>
+            </Stack>
+          )}
+          {/* {discord && ( */}
```
Comment (fozziethebeat) [inline website/src/pages/auth/signup.tsx:39]: Question, why is this part commented out?

**Review by fozziethebeat — CHANGES_REQUESTED** [review]
Can we add in the optional hiding and displaying of the discord button?  we can also do the same with `github` and I'll soon be adding in dev credentials for that method.

File: website/src/pages/auth/signup.tsx (hunk 1 above)
Comment (rsandb) [reply website/src/pages/auth/signup.tsx:39]: Just to show the Discord button. It wasn't appearing with that in place, so I commented it out while I was formatting everything

**Review by fozziethebeat — APPROVED** [review]
nice!

**Pull request merged**